org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/IScannableDeviceService.java

```diff
@@ -72,7 +75,50 @@
 	 * @throws ScanningException if no scannable with the given name could be found
 	 */
 	<T> IScannable<T> getScannable(String name) throws ScanningException;
-
+	
+	/**
+	 * Returns a list of the scannables for the given {@link Iterable} over {@link IPosition}s.
+	 * @param positionIterable an iterator over positions
+	 * @return list of scannables for the given position iterable
+	 * @throws ScanningException if one or more of the scannables cannot be found
+	 */
+	default List<IScannable<?>> getScannables(Iterable<IPosition> positionIterable) throws ScanningException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)